INSTALL.md

```diff
@@ -30,9 +30,10 @@ If you'd like to create a VM with BLT, you will require the following additional
 
         brew tap caskroom/cask
         brew install ansible
+        brew upgrade ansible
```

Comment:
Hm. You probably needed to update brew itself first to get the latest formulaes.